pygmt/figure.py

```diff
@@ -401,89 +401,98 @@ def savefig(  # noqa: PLR0912
+        it in the default viewer for your operating system (falls back to the default
+        web browser).
 
-        :func:`pygmt.set_display` can select the default display method
-        (``"notebook"``, ``"external"``, ``"none"`` or ``None``).
+        :func:`pygmt.set_display` can select the default display method (``"notebook"``,
+        ``"external"``, ``"none"`` or ``None``).
 
-        The ``method`` parameter can also override the default display method
-        for the current figure. Parameters ``dpi`` and ``width`` can be used
-        to control the resolution and dimension of the figure in the notebook.
+        The ``method`` parameter can also override the default display method for the
+        current figure. Parameters ``dpi`` and ``width`` can be used to control the
+        resolution and dimension of the figure in the notebook.
 
-        **Note**: The external viewer can be disabled by setting the
-        PYGMT_USE_EXTERNAL_DISPLAY environment variable to **false**.
-        This is useful when running unit tests and building the documentation
-        in consoles without a Graphical User Interface.
+        The external viewer can be disabled by setting environment variable
+        **PYGMT_USE_EXTERNAL_DISPLAY** to ``"false"``. This is useful when running tests
+        and building the documentation to avoid popping up windows.
 
-        Note that the external viewer does not block the current process, thus
-        it's necessary to suspend the execution of the current process for a
-        short while after launching the external viewer, so that the preview
-        image won't be deleted before the external viewer tries to open it. Set
-        the ``waiting`` parameter to a larger number if your computer is slow.
+        The external viewer does not block the current process, thus it's necessary to
+        suspend the execution of the current process for a short while after launching
+        the external viewer, so that the preview image won't be deleted before the
+        external viewer tries to open it. Set the ``waiting`` parameter to a larger
+        number if the image viewer on your computer is slow to open a file.
 
         Parameters
         ----------
-        dpi : int
-            The image resolution (dots per inch) in Jupyter notebooks.
-        width : int
-            The image width (in pixels) in Jupyter notebooks.
-        method : str or None
-            How the current figure will be displayed. Choose from:
+        method
+            The method to display the current image preview. Choose from:
 
             - ``"external"``: External PDF preview using the default PDF viewer
             - ``"notebook"``: Inline PNG preview in the current notebook
             - ``"none"``: Disable image preview
-            - ``None``: Reset to the default display method
+            - ``None``: Reset to the default display method.
```

Comment:
If we add a period here we should do the same for the other bullet points to have this consistent.

Reply:
Reading this post (https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists), I feel the period is unnecessary.

Reply:
Yep, I actually also prefer to remove it 🙂.